src/main/groovy/com/vanniktech/android/javadoc/extensions/Options.groovy

```diff
@@ -0,0 +1,37 @@
+package com.vanniktech.android.javadoc.extensions
+
+import org.gradle.api.Project
+
+/**
+ * <p>Created on 23/06/17
+ *
+ * @author Bastien Paul
+ */
+public class Options {
+    /**
+     * Closure used for filter some variant out
+     */
+    Closure variantFilter = { variant ->
+        if (variant) {
+            return true
+        } else {
+            return false
+        }
+    }
+
+    /**
+     * Closure used for customizing the task name generation.
+     * <p> It is always following the pattern : generate{{ClosureOutput}.capitalize()}Javadoc
+     */
+    Closure taskNameTransformer = { variant ->
+        return variant.name
+    }
+
+    /**
+     * Closure used for changing the output folder of the documentation
+     */
+    Closure outputDir = { Project project ->
+        "${project.buildDir}/docs/javadoc/"
+    }
+
```

Comment:
I don't want to remove this one because I would like it to be the default. By default, javadoc generated by java plugin is stored in build/docs/javadoc.